webserver: implement support for Linux abstract namespace sockets (#16613)
PowerDNS/pdns: docs/settings.rst

-  Default: 127.0.0.1,::1

Webserver/API access is only allowed from these subnets.

> Ignored if ``webserver-address`` is set to a UNIX domain socket or abstract namespace.

Reviewer: Similarly,
```suggestion
Ignored unless ``webserver-address`` is set to an IP address.
```
